### PR TITLE
Refactor Dialog with Aphrodite

### LINKED ISSUE #8984: Refactor Dialog with Aphrodite
**Describe the issue you encountered:** Refactor Dialog with Aphrodite.

- Extra QA steps:
    1. Open `Import Browser Data...` from Bookmarks on the main menu
    2. Open devtool
    3. Make sure `dialog` is styled properly

- Screenshot if needed:

- Any related issues: 2dd92a9

### PR DESCRIPTION
**Pull request by luixxiul**
Closes #8984

Per https://github.com/brave/browser-laptop/commit/2dd92a9eb3b42053ceaafc32a63e1d3508e8c039, dialog.less can be safely removed (as styles under 'menu' has been just copied).

Test Plan:
1. Open 'Import Browser Data...' from Bookmarks on the main menu
2. Open devtool
3. Make sure 'dialog' is styled properly

Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Reviewer Checklist:

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

File: app/renderer/components/common/dialog.js [hunk 1]
```diff
@@ -40,6 +43,7 @@ class Dialog extends ImmutableComponent {
   render () {
     return <div className={cx({
       dialog: true,
```
Comment (cezaraugusto) [inline app/renderer/components/common/dialog.js]: is it needed somewhere? we are removing style class and I can't see any test using it.

File: less/dialogs.less [hunk 2]
```diff
@@ -1,85 +0,0 @@
...
-        list-style-type: none;
-      }
-
-      ul.select li:hover {
-        background: #008FFF;
-        cursor: pointer;
-      }
-    }
-  }
-}
```
Comment (cezaraugusto) [inline less/dialogs.less]: I can't get enough of less less :)

**Comment by cezaraugusto** [comment]
I would recommend removing `.dialog` class since we're not using as far I can see. Other than that LGTM.

File: app/renderer/components/common/dialog.js (hunk 1 above)
Comment (luixxiul) [inline app/renderer/components/common/dialog.js]: oops I just forgot to check that out. I'm going to remove it, thanks.

Comment (luixxiul) [inline app/renderer/components/common/dialog.js]: Fixed with 3ba6110

**Comment by luixxiul** [comment]
My bad, that conversion breaks the bookmarkHanger (which I often forget that it is also a component with Dialog). Fortunately the change is not available on 0.15.300 series, so I'm fixing it before 0.16.100. thanks!

**Pull request merged**